releasenotes/notes/0.25/discrete-pulse-library-deprecation-3a95eba7e29d8d49.yaml

```diff
@@ -1,14 +1,16 @@
 ---
-features:
+features_pulse:
   - |
-    The :class:`~qiskit.pulse.SymbolicPulse` library was extended. The new pulses in the library are:
+
+    The :class:`~qiskit.pulse.library.SymbolicPulse` library was extended. The new pulse functions
+    in the library are:
 
       * :func:`~qiskit.pulse.library.GaussianDeriv`
       * :func:`~qiskit.pulse.library.Sech`
       * :func:`~qiskit.pulse.library.SechDeriv`
       * :func:`~qiskit.pulse.library.Square`
 
-    The new functions return a :class:`ScalableSymbolicPulse`, and match the functionality
+    The new functions return a :class:`~qiskit.pulse.library.ScalableSymbolicPulse` instance, and match the functionality
     of the corresponding functions in the discrete pulse library, with the exception of
-    `Square()` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
-    discrete `square()` where such a shift was induced by a :math:`\\pi` phase).
+    :func:`~qiskit.pulse.library.Square` for which a phase of :math:`2\\pi` shifts by a full cycle (contrary to the
+    discrete :func:`~qiskit.pulse.library.square` where such a shift was induced by a :math:`\\pi` phase).
```

Comment:
Do the `\\pi` things work here?  Feels like it should just be `\pi` - I don't think YAML treats `\` as a universal escape in `-|` strings.

Reply:
It renders like "2pi". I'll try it with `\` to see if we get the proper symbol.

Reply:
Yeah, it should look like $2\pi$ when it's working correctly.

Reply:
Fixed in 99af65b.